File: Source/AlphaTab/Audio/Synth/PlaybackRange.cs
namespace AlphaTab.Audio.Synth
{
    /// <summary>
    /// Represents a range of the song that should be played.
    /// </summary>
    public class PlaybackRange
    {
        /// <summary>
        /// The position in midi ticks from where the song should start.
        /// </summary>
        public int StartTick { get; set; }

        /// <summary>
        /// The position in midi ticks to where the song should be played.
        /// </summary>
        public int EndTick { get; set; }
    }
}

File: src/audio/synth/PlaybackRange.ts
/**
 * Represents a range of the song that should be played.
 */
export class PlaybackRange {
    /**
     * The position in midi ticks from where the song should start.
     */
    public startTick: number = 0;

    /**
     * The position in midi ticks to where the song should be played.
     */
    public endTick: number = 0;
}
